charts/aws-ebs-csi-driver/templates/node-windows.yaml

```diff
@@ -54,7 +56,7 @@ spec:
         {{- end }}
       containers:
         - name: ebs-plugin
-          image: {{ .Values.image.repository }}:{{ .Values.image.tag }}
+          image: {{ printf "%s:%s" .Values.image.repository (default (printf "v%s" .Chart.AppVersion) (toString .Values.image.tag)) }}
```

Comment:
Will .Values.image.tag be empty after change?
Can we use the v{Chart.appversion} in values.yaml continue use the tag field in the template?

Reply:
If image.tag is not set it will fall back to the value in appVersion.  When running a helm install/upgrade you can either use --values to pass a yaml file of values that could contain image.tag to provide a version or use --set image.tag to set the version to use.

https://helm.sh/docs/helm/helm_install/